src/opm/output/eclipse/Summary.cpp

```diff
@@ -2230,7 +2230,11 @@ static const std::unordered_map< std::string, Opm::UnitSystem::measure> single_v
   {"FGIPL"    , Opm::UnitSystem::measure::gas_surface_volume },
   {"FGIPG"    , Opm::UnitSystem::measure::gas_surface_volume },
   {"FPR"      , Opm::UnitSystem::measure::pressure },
-
+  {"FGCDI"    , Opm::UnitSystem::measure::liquid_surface_volume },
+  {"FGCDM"    , Opm::UnitSystem::measure::liquid_surface_volume },
+  {"FWCD"     , Opm::UnitSystem::measure::gas_surface_volume },
```

Comment:
This appears to use two different units of measurement for CO2, depending on whether it is in the gas phase (`FG*`) or in the water phase (`FW*`).  Is that intentional?

Reply:
Thanks for noticing.